src/app/shared/dialog/registration/registration.component.html

```diff
@@ -2,32 +2,32 @@
   <div [formGroup]="dialog.form" class="justify-self-center">
     <div class="form-group">
       <div class="input-group-lg">
-        <input type="text" class="form-control" name="firstName" placeholder="{{ 'SHARED.DIALOG.REGISTRATION.FIRST_NAME' | translate }}" formControlName="firstName" [readonly]="isComplete()" [autofocus]="isSubmit()" [ngClass]="{'is-valid': isSubmit() && isValid('firstName'), 'is-invalid': isInvalid('firstName')}" aria-describedby="firstNameError" aria-required="true" />
-        <div *ngIf="isInvalid('firstName')" id="firstNameError" class="invalid-feedback" role="alert">{{getErrorMessage('firstName') | async}}</div>
+        <input type="text" class="form-control" name="firstName" placeholder="{{ 'SHARED.DIALOG.REGISTRATION.FIRST_NAME' | translate }}" formControlName="firstName" [readonly]="isComplete()" [autofocus]="isSubmit()" [ngClass]="{'is-valid': isSubmit() && isValid('firstName'), 'is-invalid': isInvalid('firstName')}" aria-describedby="first-Name-Error" aria-required="true" />
+        <div *ngIf="isInvalid('firstName')" id="first-Name-Error" class="invalid-feedback" role="alert">{{getErrorMessage('firstName') | async}}</div>
```

Comment:
```suggestion
        <input type="text" class="form-control" name="firstName" placeholder="{{ 'SHARED.DIALOG.REGISTRATION.FIRST_NAME' | translate }}" formControlName="firstName" [readonly]="isComplete()" [autofocus]="isSubmit()" [ngClass]="{'is-valid': isSubmit() && isValid('firstName'), 'is-invalid': isInvalid('firstName')}" aria-describedby="first-name-error" aria-required="true" />
        <div *ngIf="isInvalid('firstName')" id="first-name-error" class="invalid-feedback" role="alert">{{getErrorMessage('firstName') | async}}</div>
```

I suspect the `N` and the `E` should not be upper case here.